.travis.yml

```diff
@@ -57,7 +57,7 @@ jobs:
       python: "3.12"
 
 install:
-  - pip install pylint coveralls pyfakefs
+  - pip install pylint coveralls pyfakefs keyring
```

Comment:
I have no idea why this comes up know but worked in the past. Here TravisCI was not able to `import keyring` so I install it now.